mysql-test/include/rpl_ddl_high_priority.inc

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+# Test replication of the high_priority syntax
+###############################################################################
+
+
+source include/have_binlog_format_statement.inc;
+--source include/master-slave.inc
+
+--source include/rpl_connection_master.inc
+
+create table t1 (i int);
+insert into t1 values (1), (2), (3);
+
+--source include/sync_slave_sql_with_master.inc
+
+--source include/rpl_connection_slave.inc
+select * from t1;
+
+--source include/rpl_connection_slave.inc
```

Comment:
This needs to be another connection to the slave. It gets killed by high priority ddl and the subsequent sync_slave_with_master call on it will fail.